lib/DigestSender.php

```diff
@@ -103,11 +110,21 @@ private function getLastSendActivity(string $user, int $now): int {
 		return $this->data->getFirstActivitySince($user, $now - (24 * 60 * 60));
 	}
 
-	public function sendDigestForUser(string $uid, int $now, string $timezone, string $language) {
+	public function updateLastSentForUser(IUser $user, int $now): void {
+		$uid = $user->getUID();
+		$lastSend = $this->getLastSendActivity($uid, $now);
+
+		['count' => $count, 'max' => $lastActivityId] = $this->data->getActivitySince($uid, $lastSend, true);
```

Comment:
```suggestion
		['max' => $lastActivityId] = $this->data->getActivitySince($uid, $lastSend, true);
```
`$count` is not used